src/promptflow/promptflow/_sdk/_submitter/experiment_orchestrator.py

```diff
@@ -81,7 +81,8 @@ def __init__(self, client, experiment: Experiment = None):
         self._node_runs = {}
 
     def test(
-        self, flow: Union[str, Path], template: ExperimentTemplate, inputs=None, environment_variables=None, **kwargs
+        self, template: ExperimentTemplate, flow: Union[str, Path] = None, inputs=None, environment_variables=None,
```

Comment:
Maybe we can have a new api for the skip scenario

Reply:
using _test_with_ui to do this

Reply:
use _test_with_ui api to handle skip/overrite unbinding flow input